python/ray/experimental/test/async_test.py

```diff
@@ -9,6 +10,7 @@
 
 @pytest.fixture
 def init():
+    os.environ["RAY_FORCE_DIRECT"] = "0"
```

Comment:
I remember from an offline conversation this flag doesn't work anymore?

Reply:
It won't work soon, but @edoakes  PR is not merged yet so until then it is still necessary